rllib/tests/test_node_failure.py

```diff
@@ -0,0 +1,226 @@
+# This workload tests RLlib's ability to recover from failing workers nodes
+import time
+import unittest
+import threading
+from ray.exceptions import RayActorError
+
+import ray
+from ray.cluster_utils import Cluster
+from ray._private.test_utils import get_other_nodes
+
+from ray.rllib.algorithms.ppo import PPO, PPOConfig
+
+num_redis_shards = 5
+redis_max_memory = 10 ** 8
+object_store_memory = 10 ** 8
+num_nodes = 3
```

Comment:
constants should be ALL_CAP?

Reply:
I'm with you, but these constants occur in many core tests and are always not capitalized.
I think consistency > general convention here, right?